2021-09-10 21:09:59,421UTC: WARNING:root:Script C:/Users/Rob/source/repos/Automation/labs/lab_utils.py cannot be run independently from this application.
2021-09-10 21:10:54,400UTC: WARNING:root:Script C:/Users/Rob/source/repos/Automation/labs/lab_utils.py cannot be run independently from this application.
2021-09-10 21:33:20,408UTC: WARNING:root:Script /home/gns3demo/Automation/labs/lab_utils.py cannot be run independently from this application.
2021-09-11 16:19:57,877UTC: WARNING:root:Script C:/Users/Rob/source/repos/Automation/labs/lab_utils.py cannot be run independently from this application.
2021-09-11 16:20:37,173UTC: WARNING:root:Script C:/Users/Rob/source/repos/Automation/labs/lab_utils.py cannot be run independently from this application.
2021-09-23 14:59:54,746UTC: CRITICAL:root:
Type RuntimeError: Unable to execute command: [sudo] password for gns3user: 
Sorry, try again.
[sudo] password for gns3user: 
Sorry, try again.
[sudo] password for gns3user: 
sudo: 3 incorrect password attempts
 in /home/gns3user/Automation/labs/test.py at line 37.

2021-09-23 15:03:05,434UTC: CRITICAL:root:
Type RuntimeError: Unable to execute command:  in /home/gns3user/Automation/labs/test.py at line 37.

2021-09-23 15:04:49,637UTC: CRITICAL:root:
Type TypeError: string indices must be integers, not str in /home/gns3user/Automation/labs/test.py at line 38.

2021-09-23 15:05:52,009UTC: CRITICAL:root:
Type RuntimeError: Unable to execute command:  in /home/gns3user/Automation/labs/test.py at line 38.

2021-09-23 15:06:40,283UTC: CRITICAL:root:
Type RuntimeError: Unable to execute command:  in /home/gns3user/Automation/labs/test.py at line 38.

2021-09-23 15:08:18,693UTC: CRITICAL:root:
Type RuntimeError: Unable to execute command: sudo: 3 incorrect password attempts in /home/gns3user/Automation/labs/test.py at line 38.

2021-09-23 15:21:53,085UTC: CRITICAL:root:
Type RuntimeError: Unable to execute command: [sudo] password for gns3user: 
Sorry, try again.
[sudo] password for gns3user: 
Sorry, try again.
[sudo] password for gns3user: 
sudo: 3 incorrect password attempts in /home/gns3user/Automation/labs/test.py at line 38.

2021-09-23 15:23:02,216UTC: CRITICAL:root:
Type RuntimeError: Unable to execute command: sudo mkdir --parents /var/lib/tftpboot: [sudo] password for gns3user: 
Sorry, try again.
[sudo] password for gns3user: 
Sorry, try again.
[sudo] password for gns3user: 
sudo: 3 incorrect password attempts in /home/gns3user/Automation/labs/test.py at line 37.

2021-09-23 17:39:08,304UTC: CRITICAL:root:
Type TIMEOUT: Expected "success", found Sorry, try again. in /home/gns3user/Automation/labs/lab000-combo.py at line 85.

2021-09-23 17:40:23,511UTC: WARNING:root:Script /home/gns3user/Automation/labs/lab_utils.py cannot be run independently from this application.
2021-09-23 17:40:38,594UTC: CRITICAL:root:
Type TypeError: object of type 'NoneType' has no len() in /home/gns3user/Automation/labs/lab000-combo.py at line 92.

2021-09-23 17:46:01,439UTC: CRITICAL:root:
Type RuntimeError: Cannot connect to console port. Ensure device is started or reloaded. in /home/gns3user/Automation/labs/lab000-combo.py at line 95.

2021-09-23 17:49:26,774UTC: CRITICAL:root:
Type KeyboardInterrupt:  in /home/gns3user/Automation/labs/lab000-combo.py at line 94.

2021-09-23 17:54:09,035UTC: CRITICAL:root:
Type RuntimeError: Cannot connect to console port. Ensure device is started or reloaded. in /home/gns3user/Automation/labs/lab000-combo.py at line 95.

2021-09-23 17:54:45,448UTC: CRITICAL:root:
Type RuntimeError: Cannot connect to console port. in /home/gns3user/Automation/labs/lab000-combo.py at line 95.

2021-09-23 17:57:46,356UTC: CRITICAL:root:
Type KeyboardInterrupt:  in /home/gns3user/Automation/labs/lab000-combo.py at line 90.

2021-09-23 18:00:19,905UTC: CRITICAL:root:
Type TypeError: 'NoneType' object is not iterable in /home/gns3user/Automation/labs/lab000-combo.py at line 90.

2021-09-23 18:02:04,033UTC: CRITICAL:root:
Type KeyboardInterrupt:  in /home/gns3user/Automation/labs/lab000-combo.py at line 94.

2021-09-23 18:05:19,611UTC: CRITICAL:root:
Type TIMEOUT: Expected "R1>", found "" in /home/gns3user/Automation/labs/lab000-combo.py at line 114.

2021-09-23 18:08:38,481UTC: CRITICAL:root:
Type TIMEOUT: Expected "R1>", found "" in /home/gns3user/Automation/labs/lab000-combo.py at line 329.

2021-09-23 18:13:50,295UTC: CRITICAL:root:
Type TypeError: 'NoneType' object is not iterable in /home/gns3user/Automation/labs/lab000-combo.py at line 90.

2021-09-23 18:16:33,856UTC: CRITICAL:root:
Type KeyboardInterrupt:  in /home/gns3user/Automation/labs/lab000-combo.py at line 90.

2021-09-23 18:17:00,362UTC: CRITICAL:root:
Type KeyboardInterrupt:  in /home/gns3user/Automation/labs/lab000-combo.py at line 90.

2021-09-23 18:17:43,448UTC: CRITICAL:root:
Type KeyboardInterrupt:  in /home/gns3user/Automation/labs/lab000-combo.py at line 91.

2021-09-23 18:29:43,438UTC: CRITICAL:root:
Type ValueError: Single '}' encountered in format string in /home/gns3user/Automation/labs/lab000-combo.py at line 154.

2021-09-23 21:11:35,188UTC: CRITICAL:root:
Type TIMEOUT: Expected "R1>", found "" in /home/gns3user/Automation/labs/lab000-combo.py at line 335.

2021-09-23 21:57:05,843UTC: CRITICAL:root:
Type TIMEOUT: Expected "R1>", found "" in lab000-combo.py at line 118.

2021-09-23 21:59:45,754UTC: CRITICAL:root:
Type TIMEOUT: Expected "R1>", found "" in lab000-combo.py at line 335.

2021-09-23 22:02:21,084UTC: CRITICAL:root:
Type TIMEOUT: Expected "R1>", found "" in lab000-combo.py at line 118.

2021-09-27 18:29:35,298UTC: CRITICAL:root:
Type RuntimeError: Cannot connect to console port. in /home/gns3user/Automation/labs/lab000-combo.py at line 99.

2021-09-27 18:33:07,839UTC: CRITICAL:root:
Type KeyboardInterrupt:  in /home/gns3user/Automation/labs/lab000-combo.py at line 86.

2021-09-27 20:50:13,400UTC: WARNING:root:Script /home/gns3user/Automation/labs/lab_utils.py cannot be run independently from this application.
2021-09-27 21:10:15,551UTC: CRITICAL:root:
Type IndexError: list index out of range in /home/gns3user/Automation/labs/test.py at line 70.

2021-09-27 21:13:09,572UTC: CRITICAL:root:
Type IndexError: list index out of range in /home/gns3user/Automation/labs/test.py at line 70.

2021-09-27 21:13:52,941UTC: CRITICAL:root:
Type IndexError: list index out of range in /home/gns3user/Automation/labs/test.py at line 70.

2021-09-27 21:15:15,789UTC: CRITICAL:root:
Type TypeError: 'NoneType' object has no attribute '__getitem__' in /home/gns3user/Automation/labs/test.py at line 70.

2021-09-27 21:22:24,713UTC: CRITICAL:root:
Type IndexError: list index out of range in /home/gns3user/Automation/labs/test.py at line 70.

2021-09-27 21:22:48,766UTC: CRITICAL:root:
Type IndexError: list index out of range in /home/gns3user/Automation/labs/test.py at line 70.

2021-09-27 21:34:33,046UTC: CRITICAL:root:
Type IndexError: list index out of range in /home/gns3user/Automation/labs/test.py at line 70.

2021-09-27 21:41:49,653UTC: CRITICAL:root:
Type IndexError: list index out of range in /home/gns3user/Automation/labs/test.py at line 71.

2021-09-27 21:41:57,359UTC: CRITICAL:root:
Type IndexError: list index out of range in /home/gns3user/Automation/labs/test.py at line 71.

2021-09-27 21:44:44,363UTC: CRITICAL:root:
Type IndexError: list index out of range in /home/gns3user/Automation/labs/test.py at line 72.

2021-09-27 21:51:38,678UTC: CRITICAL:root:
Type TypeError: expand() takes exactly one argument (0 given) in /home/gns3user/Automation/labs/test.py at line 71.

2021-09-27 22:07:28,668UTC: CRITICAL:root:
Type TIMEOUT: Expected re.compile(".+"), found "" in /home/gns3user/Automation/labs/lab000-combo.py at line 122.

2021-09-27 22:10:47,522UTC: CRITICAL:root:
Type TIMEOUT: Expected re.compile(".+"), found "" in /home/gns3user/Automation/labs/lab000-combo.py at line 122.

2021-09-27 22:27:50,202UTC: CRITICAL:root:
Type TIMEOUT: Expected EOF, found "OS Software, 3700 Software (C3745-ADVENTERPRISEK9-M), Version 12.4(25d), RELEASE SOFTWARE (fc1)" in /home/gns3user/Automation/labs/lab000-combo.py at line 124.

2021-09-27 22:29:01,049UTC: CRITICAL:root:
Type TIMEOUT: Expected EOF, found "OS Software, 3700 Software (C3745-ADVENTERPRISEK9-M), Version 12.4(25d), RELEASE SOFTWARE (fc1)" in /home/gns3user/Automation/labs/lab000-combo.py at line 124.

2021-09-27 22:35:16,748UTC: CRITICAL:root:
Type TypeError: not all arguments converted during string formatting in /home/gns3user/Automation/labs/lab000-combo.py at line 125.

2021-09-27 22:35:52,773UTC: CRITICAL:root:
Type TypeError: not all arguments converted during string formatting in /home/gns3user/Automation/labs/lab000-combo.py at line 125.

2021-09-27 22:42:51,398UTC: CRITICAL:root:
Type TIMEOUT: Expected "oftware, found "OS Software, 3700 Software (C3745-ADVENTERPRISEK9-M), Version 12.4(25d), RELEASE SOFTWARE (fc1)" in /home/gns3user/Automation/labs/lab000-combo.py at line 124.

2021-09-27 22:53:53,376UTC: CRITICAL:root:
Type TIMEOUT: Expected re.compile("[Ss]oftware"), found "" in /home/gns3user/Automation/labs/lab000-combo.py at line 125.

2021-09-27 22:54:57,424UTC: CRITICAL:root:
Type TIMEOUT: Expected re.compile("[Ss]oftware"), found "ftware ; Version" in /home/gns3user/Automation/labs/lab000-combo.py at line 125.

2021-09-27 23:09:10,936UTC: CRITICAL:root:
Type TIMEOUT: Expected EOF, found ".986: %SYS-5-CONFIG_I: Configured from console by console" in /home/gns3user/Automation/labs/lab000-combo.py at line 122.

2021-09-27 23:15:12,515UTC: CRITICAL:root:
Type TypeError: 'str' object is not callable in /home/gns3user/Automation/labs/lab000-combo.py at line 121.

2021-09-27 23:43:04,114UTC: CRITICAL:root:
Type TIMEOUT: Expected "success", found "Sorry, try again." in /home/gns3user/Automation/labs/lab000-combo.py at line 88.

2021-09-29 16:26:11,522UTC: CRITICAL:root:
Type TIMEOUT: Expected re.compile("include [IOSios] [Ss]oftware"), found "OS Software, 3700 Software (C3745-ADVENTERPRISEK9-M), Version 12.4(25d), RELEASE SOFTWARE (fc1)" in /home/gns3user/Automation/labs/lab000-combo.py at line 134.

2021-09-29 16:28:24,936UTC: CRITICAL:root:
Type TIMEOUT: Expected re.compile("include [IOSios] [Ss]oftware"), found "OS Software, 3700 Software (C3745-ADVENTERPRISEK9-M), Version 12.4(25d), RELEASE SOFTWARE (fc1)" in /home/gns3user/Automation/labs/lab000-combo.py at line 134.

2021-09-29 16:42:46,509UTC: CRITICAL:root:
Type TIMEOUT: Expected EOF, found "OS Software, 3700 Software (C3745-ADVENTERPRISEK9-M), Version 12.4(25d), RELEASE SOFTWARE (fc1)" in /home/gns3user/Automation/labs/lab000-combo.py at line 135.

2021-09-29 16:50:27,672UTC: CRITICAL:root:
Type TIMEOUT: Expected EOF, found "OS Software, 3700 Software (C3745-ADVENTERPRISEK9-M), Version 12.4(25d), RELEASE SOFTWARE (fc1)" in /home/gns3user/Automation/labs/lab000-combo.py at line 135.

2021-09-29 17:05:19,506UTC: CRITICAL:root:
Type TIMEOUT: Expected EOF, found "OS Software, 3700 Software (C3745-ADVENTERPRISEK9-M), Version 12.4(25d), RELEASE SOFTWARE (fc1)" in /home/gns3user/Automation/labs/lab000-combo.py at line 136.

2021-09-29 17:26:43,266UTC: CRITICAL:root:
Type TIMEOUT: Expected EOF, found "OS Software, 3700 Software (C3745-ADVENTERPRISEK9-M), Version 12.4(25d), RELEASE SOFTWARE (fc1)" in /home/gns3user/Automation/labs/lab000-combo.py at line 134.

2021-09-29 17:48:37,406UTC: CRITICAL:root:
Type TIMEOUT: Expected "subcommands, found " unable to find computer address" in /home/gns3user/Automation/labs/lab000-combo.py at line 132.

2021-09-29 17:50:33,362UTC: CRITICAL:root:
Type TIMEOUT: Expected "subcommands, found "le to find computer address" in /home/gns3user/Automation/labs/lab000-combo.py at line 133.

2021-09-29 18:25:07,971UTC: CRITICAL:root:
Type TIMEOUT: Expected "R1#, found "5/5), round-trip min/avg/max = 32/34/36 ms" in /home/gns3user/Automation/labs/lab000-combo.py at line 135.

2021-09-29 18:34:56,039UTC: CRITICAL:root:
Type TIMEOUT: Expected "R1#, found "5/5), round-trip min/avg/max = 32/35/40 ms" in /home/gns3user/Automation/labs/lab000-combo.py at line 129.

2021-09-29 19:00:56,414UTC: CRITICAL:root:
Type TIMEOUT: Expected "R1#, found "ent (5/5), round-trip min/avg/max = 32/32/36 ms" in /home/gns3user/Automation/labs/lab000-combo.py at line 129.

2021-09-29 19:01:38,546UTC: CRITICAL:root:
Type IndexError: list index out of range in /home/gns3user/Automation/labs/lab000-combo.py at line 132.

2021-09-29 19:05:48,439UTC: CRITICAL:root:
Type KeyboardInterrupt:  in /home/gns3user/Automation/labs/lab000-combo.py at line 133.

2021-09-29 19:07:08,958UTC: CRITICAL:root:
Type KeyboardInterrupt:  in /home/gns3user/Automation/labs/lab000-combo.py at line 133.

2021-09-29 19:08:00,524UTC: CRITICAL:root:
Type RuntimeError: Cannot connect to console port. in /home/gns3user/Automation/labs/lab000-combo.py at line 104.

2021-09-29 19:08:48,387UTC: CRITICAL:root:
Type RuntimeError: Cannot connect to console port. in /home/gns3user/Automation/labs/lab000-combo.py at line 104.

2021-09-29 19:11:20,610UTC: CRITICAL:root:
Type TIMEOUT: Expected "R1#, found "" in /home/gns3user/Automation/labs/lab000-combo.py at line 129.

2021-09-29 19:16:12,109UTC: CRITICAL:root:
Type KeyboardInterrupt:  in /home/gns3user/Automation/labs/lab000-combo.py at line 133.

2021-09-29 19:30:58,733UTC: CRITICAL:root:
Type RuntimeError: Cannot get the device's name. in /home/gns3user/Automation/labs/lab000-combo.py at line 144.

2021-09-29 19:33:57,189UTC: CRITICAL:root:
Type RuntimeError: "Cannot get the device's name." in /home/gns3user/Automation/labs/lab000-combo.py at line 144.

2021-09-29 20:54:34,919UTC: CRITICAL:root:
Type IndexError: "list index out of range" in /home/gns3user/Automation/labs/lab000-combo.py at line 149.

2021-09-30 00:46:14,726UTC: CRITICAL:root:
Type RuntimeError: "Unable to sudo systemctl start tftp: sudo: no tty present and no askpass program specified" in /home/gns3user/Automation/labs/lab000-combo.py at line 269.

2021-09-30 15:11:18,524UTC: CRITICAL:root:
Type TypeError: "__prompt_for_password() takes no arguments (1 given)" in /home/gns3user/Automation/labs/test.py at line 240.

2021-09-30 15:15:52,713UTC: CRITICAL:root:
Type RuntimeError: "Unable to sudo systemctl stop tftp: sudo: no tty present and no askpass program specified" in /home/gns3user/Automation/labs/test.py at line 64.

2021-09-30 15:51:59,425UTC: CRITICAL:root:
Type ExceptionPexpect: "The command was not found or was not executable: booga." in /home/gns3user/Automation/labs/test.py at line 219.

2021-09-30 15:57:28,943UTC: CRITICAL:root:
Type RuntimeError: "Unable to systemctl stop tftp: Failed to stop tftp.service: Access denied
See system logs and 'systemctl status tftp.service' for details.
Warning: Stopping tftp.service, but it can still be activated by:
  tftp.socket" in /home/gns3user/Automation/labs/test.py at line 64.

2021-09-30 18:05:55,481UTC: CRITICAL:root:
Type RuntimeError: "Cannot download startup-config from device using TFTP." in /home/gns3user/Automation/labs/lab000-combo.py at line 283.

2021-09-30 18:13:31,761UTC: CRITICAL:root:
Type RuntimeError: "Cannot download startup-config from device using TFTP." in /home/gns3user/Automation/labs/lab000-combo.py at line 283.

2021-09-30 18:20:40,877UTC: CRITICAL:root:
Type RuntimeError: "Cannot download startup-config from device using TFTP." in /home/gns3user/Automation/labs/lab000-combo.py at line 283.

2021-09-30 18:47:07,856UTC: CRITICAL:root:
Type RuntimeError: "Unable to sudo systemctl stop tftp: sudo: no tty present and no askpass program specified" in /home/gns3user/Automation/labs/test.py at line 71.

2021-09-30 18:51:43,028UTC: CRITICAL:root:
Type RuntimeError: "Unable to sudo systemctl stop tftp: sudo: no tty present and no askpass program specified" in /home/gns3user/Automation/labs/test.py at line 71.

2021-09-30 18:54:25,965UTC: CRITICAL:root:
Type RuntimeError: "Unable to sudo systemctl stop tftp: sudo: no tty present and no askpass program specified" in /home/gns3user/Automation/labs/test.py at line 73.

2021-09-30 18:59:27,858UTC: CRITICAL:root:
Type RuntimeError: "Unable to sudo firewall-cmd --zone=public --add-service=tftp: sudo: no tty present and no askpass program specified" in /home/gns3user/Automation/labs/test.py at line 58.

2021-09-30 18:59:45,297UTC: CRITICAL:root:
Type RuntimeError: "Unable to sudo -S firewall-cmd --zone=public --add-service=tftp: [sudo] password for gns3user: 
sudo: no password was provided" in /home/gns3user/Automation/labs/test.py at line 58.

2021-09-30 19:15:06,151UTC: CRITICAL:root:
Type KeyboardInterrupt: "" in /home/gns3user/Automation/labs/test.py at line 31.

2021-09-30 19:20:42,954UTC: CRITICAL:root:
Type RuntimeError: "Cannot download startup-config from device using TFTP." in /home/gns3user/Automation/labs/lab000-combo.py at line 283.

2021-09-30 19:30:41,254UTC: CRITICAL:root:
Type RuntimeError: "Cannot download startup-config from device using TFTP." in /home/gns3user/Automation/labs/lab000-combo.py at line 285.

2021-09-30 19:32:01,989UTC: CRITICAL:root:
Type KeyboardInterrupt: "" in /home/gns3user/Automation/labs/lab000-combo.py at line 90.

2021-09-30 19:33:26,882UTC: CRITICAL:root:
Type RuntimeError: "Cannot download startup-config from device using TFTP." in /home/gns3user/Automation/labs/lab000-combo.py at line 286.

2021-09-30 20:05:58,496UTC: CRITICAL:root:
Type RuntimeError: "Cannot connect to console port." in /home/gns3user/Automation/labs/lab000-combo.py at line 113.

2021-09-30 20:11:44,848UTC: CRITICAL:root:
Type KeyboardInterrupt: "" in /home/gns3user/Automation/labs/lab000-combo.py at line 90.

2021-09-30 20:15:29,326UTC: CRITICAL:root:
Type IndexError: "tuple index out of range" in /home/gns3user/Automation/labs/lab000-combo.py at line 270.

2021-09-30 20:25:07,496UTC: CRITICAL:root:
Type RuntimeError: "Cannot download startup-config from device using TFTP." in /home/gns3user/Automation/labs/lab000-combo.py at line 284.

2021-09-30 20:29:22,751UTC: CRITICAL:root:
Type TIMEOUT: "Expected "Destination filename", found "ads""" in /home/gns3user/Automation/labs/lab000-combo.py at line 279.

2021-09-30 20:34:03,383UTC: CRITICAL:root:
Type RuntimeError: "Cannot download startup-config from device using TFTP." in /home/gns3user/Automation/labs/lab000-combo.py at line 283.

